test/cel-validation/backendtrafficpolicy_test.go

```diff
@@ -174,6 +174,53 @@ func TestBackendTrafficPolicyTarget(t *testing.T) {
 			},
 			wantErrors: []string{},
 		},
+		{
+			desc: "consistentHash header field not nil when consistentHashType is header",
```

Comment:
I think this is where the validation tests are supposed to go, but when I run `make test` this directory is not listed in the output. I only see:

```
> make test
...
?       github.com/envoyproxy/gateway/internal/xds/utils        [no test files]
?       github.com/envoyproxy/gateway/test/e2e/tests    [no test files]
?       github.com/envoyproxy/gateway/proto/extension   [no test files]
...
```

So I'm not convinced they're passing yet. Is there a CI pipeline or different command I should be running?

Reply:
`make help` can come in handy here to find the right target
* `make tube-test` 
* `make go.test.cel`

should run the tests with the `cel` tag